### DISCUSSION TITLE
A way to accept all incoming/local hunks in merge conflicts for files/folders?

### DISCUSSION THREAD

**Discussion by DominickVale** [jesseduffield/lazygit#2052] [Q&A]
Like in jetbrains editors, where you can right click on a folder/file in the git panel and select whether to accept all local or incoming changes, instead of having to stage all hunks manually. I saw there is a "accept all hunks" functionality, but it only accepts both incoming and local... So is there a way to do it currently?

**Comment by mark2185** [comment]
Currently only through a custom command, see the discussion [here](https://github.com/jesseduffield/lazygit/issues/2026).

**Reply by DominickVale** [reply]
Thank you. It works on files, but it doesn't seem to work on folders...

**Reply by mark2185** [reply]
Yeah, that's discussed [here](https://github.com/jesseduffield/lazygit/issues/2003). I've added your requirement to that issue as to better keep track of it.
